ops/model.py

```diff
@@ -1911,7 +1923,8 @@ def from_name(cls, name: str, message: str):
         does not have an associated message.
 
         Args:
-            name: Name of the status, for example "active" or "blocked".
+            name: Name of the status, one of:
+                "active", "blocked", "maintenance", "waiting", "error", or "unknown".
```

Comment:
I think we should revert this change, as the range of values is now shown in the type, [for example](https://ops--1354.org.readthedocs.build/en/1354/#ops.StatusBase.name)

Reply:
Nah, I'd missed which `name` this was - ignore.